src/logger/logger.module.ts

```diff
@@ -0,0 +1,11 @@
+import { Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { ExceptionLogEntity } from './entities/exception-log.entity';
+import { ExceptionLogService } from './exception-log/exception-log.service';
+
+@Module({
+  imports: [TypeOrmModule.forFeature([ExceptionLogEntity])],
```

Comment:
It's not correct. use the `ExceptionLogRepository`.
Did you see class film?

Reply:
Ok, done.